app/models/concerns/iiif_print/set_child_flag.rb

```diff
@@ -4,6 +4,10 @@ module RDF
   class CustomIsChildTerm < Vocabulary('http://id.loc.gov/vocabulary/identifiers/')
```

Comment:
maybe consider renaming this file since it's doing more than setting the child flag now. I suppose that would be a breaking change so we could also cut a new release? 

Now that I say that, maybe that can be a TODO for later given the urgency 😅

Reply:
lol yeah I considered that but since it's included in EVERY repo that uses IiifPrint, it's def a breaking change which is why I left it. Plus it's still a type of child flag if you stretch hard enough. ;)